graphql/database/groupImpact/ImpactMetricModel.js

```diff
@@ -0,0 +1,65 @@
+import DynamoDBModel from '../base/DynamoDBModel'
+import types from '../fieldTypes'
+import { IMPACT_METRIC } from '../constants'
+
+/*
+ * @extends DynamoDBModel
+ */
+class ImpactMetric extends DynamoDBModel {
+  static get name() {
+    return IMPACT_METRIC
+  }
+
+  static get hashKey() {
+    return 'id'
+  }
+
+  static get tableName() {
```

Comment:
Just confirming: we'll be drawing from JSON/YML data for the near term, right? No need for this to live in a DB right now.